### PR TITLE
Add cancel bid

### PR DESCRIPTION
**Pull request by JamesEarle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,6 @@
   },
   "peerDependencies": {
     "ethers": "^5.4.6",
-    "@zero-tech/zauction-sdk": ">=0.0.26"
+    "@zero-tech/zauction-sdk": ">=0.0.28"
```
Comment (MattCharles) [inline package.json:50]: should we pin all dependencies as a rule? Obviously we control this one so could limit blast radius but ethers right there has a ^ on it. is there a way to get a linter or something to do this for us

**Review by MattCharles — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (JamesEarle) [reply package.json:50]: We do pin all dependencies as a rule, I just keep forgetting to do that. Would be great as a linter feature, I'll see if I can work that out

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "graphql": "15.5.3"
   },
   "peerDependencies": {
-    "ethers": "^5.4.6",
```
Comment (Remscar) [inline package.json:49]: Please do not pin peer dependencies.

File: package.json (hunk 1 above)
Comment (Remscar) [reply package.json:50]: Peer dependencies don't need to be pinned.

**Review by Remscar — CHANGES_REQUESTED** [review]
Peer dependency needs to be unpinned, otherwise this looks good

**Review by Remscar — APPROVED** [review]

**Pull request merged**